src/openforms/formio/tests/test_api_fileupload.py

```diff
@@ -184,6 +184,24 @@ def test_content_inconsistent_with_mime_type(self):
             status_code=status.HTTP_400_BAD_REQUEST,
         )
 
+    @tag("gh-5507")
+    def test_msg_mimetype_override(self):
+        self._add_submission_to_session(self.submission)
+        url = reverse("api:formio:temporary-file-upload")
+        with open(Path(TEST_FILES, "sample.msg"), "rb") as f:
+            file = SimpleUploadedFile(
+                "sample.msg", f.read(), content_type="application/octet-stream"
+            )
```

Comment:
I believe you can simplify this to:


```suggestion
        with (TEST_FILES / "sample.msg").open("rb") as infile:
            file = SimpleUploadedFile(
                "sample.msg", infile, content_type="application/octet-stream"
            )
```